pkg/controller/migstorage/migstorage_controller.go

```diff
@@ -139,7 +147,7 @@ func (r *ReconcileMigStorage) Reconcile(request reconcile.Request) (reconcile.Re
 	// Ready
 	storage.Status.SetReady(
 		!storage.Status.HasBlockerCondition(),
-		ReadyMessage)
+		"The storage is ready.")
```

Comment:
@jortel I was talking to @alaypatel07 and he mentioned you guys decided to move away from shared message consts to having these messages inline. 

I'd like to understand the reasoning behind this just so I'm aware.

Reply:
Mainly because it's an extra set of `const` to maintain and unlike all of the other fields are only used in (1) place. Another reason is that the Message _may_ be parameterized and keeping the string and Sprintf() together would be easier to maintain.

Reply:
Alay and I talked about this yesterday, that makes sense to me, 👍  on that change.

Reply:
resolving this. feel free to re-open if needed